src/pages/TablePage.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { useContext } from 'react'
+import { MyNoteContext } from '../context/NoteState'
+
+function TablePage() {
```

Comment:
tablepage do not represent separate page , this is part of main page which is home page.
so make it as  a component of home page.

Reply:
Resolved sir